app/views/admin/_left_hand_navigation.haml

```diff
@@ -3,7 +3,7 @@
 - links_manage = links[2..5]
 - links_configure = links[6..21]
 
-= render :partial => "admin/uservoice_widget"
+= render :partial => "admin/uservoice_widget" if APP_CONFIG.uservoice_widget_url.present?
```

Comment:
Ok, thanks for mentioning. Had not picked that practice yet, but will modify these to match current best practice.